deltacat/compute/compactor_v2/compaction_session.py

```diff
@@ -507,26 +539,24 @@ def merge_input_provider(index, item):
         and compaction_audit.hash_bucket_processed_size_bytes
     ):
         input_inflation = (
-            compaction_audit.hash_bucket_processed_size_bytes
-            / compaction_audit.input_size_bytes
-        )
+            compaction_audit.hash_bucket_processed_size_bytes + delete_file_size_bytes
```

Comment:
Why is this changed? We need to change in `task_options.py` to account for merge memory estimation.